ietf-microwave-radio-link.yang

```diff
@@ -121,6 +126,45 @@ module ietf-microwave-radio-link {
        for configuration or status.";
   }
 
+  grouping header-compression-profile {
+  leaf header-compression-profile-name {
+    type string;
+    description
+      "none";
```

Comment:
A meaningful description should be provided:

```suggestion
      "A name that uniquely identifies the header compression profile in an RLT.";
```

Note that since the name is the key of the list, it cannot be changed.

Alternatively, two data nodes can be defined (as string): an 'identifier' used as a key and a 'name' which can be configured by the operator (like a user label)